tests/test_core.py

```diff
@@ -6711,11 +6711,10 @@ def test_demangle_stacks_symbol_map(self):
 
   def test_tracing(self):
     self.emcc_args += ['--tracing']
-
     self.do_run_in_out_file_test('tests', 'core', 'test_tracing')
 
   def test_eval_ctors(self):
-    if '-O2' not in str(self.emcc_args) or '-O1' in str(self.emcc_args):
+    if not self.is_wasm_backend() and '-O2' not in str(self.emcc_args) or '-O1' in str(self.emcc_args):
```

Comment:
I'd actually prefer to keep the test running on the wasm backend for now. The code has bugs that prevent it running by default, but we hope to fix those, and should not break it meanwhile.

Reply:
I think you misread the change.  This allows the test to run also for O0 .. but only in the wasm backend, so I'm actually increasing the test coverage.   

This early return is not needed for wasm backend as far as I can tell.

Reply:
Oh, sorry, I misunderstood it then.